docs/modules/deployment/pages/core/deb-debian/postgresql.adoc

```diff
@@ -1,7 +1,22 @@
+.Install GnuPG, cURL and certificates
+[source, console]
+----
+sudo apt -y install curl gnupg ca-certificates
+----
+
+.Install the official PostgreSQL repository
+[source, console]
+----
+source /etc/os-release
+curl -fsSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo gpg --dearmor -o /usr/share/keyrings/postgresql.gpg
+echo "deb [signed-by=/usr/share/keyrings/postgresql.gpg arch=amd64] http://apt.postgresql.org/pub/repos/apt/ ${VERSION_CODENAME}-pgdg main" | sudo tee /etc/apt/sources.list.d/postgresql.list
+sudo apt update
+----
+
```

Comment:
Might be worth dropping a link to PGDG's official deb repo doc.  https://wiki.postgresql.org/wiki/Apt#Manual_Repository_Configuration